lib/linguist/languages.yml

```diff
@@ -4847,6 +4847,17 @@ POV-Ray SDL:
   tm_scope: source.pov-ray sdl
   ace_mode: text
   language_id: 275
+Pact:
+  type: programming
+  color: "#F7A8B8"
+  ace_mode: text
+  tm_scope: source.pact
+  extensions:
+  - ".pact"
+  - ".repl"
+  interpreters:
+  - pact
```

Comment:
```suggestion
```

This field is for languages which use a shebang in some of their files. Pact doesn't appear to be one of those so we can remove this.

Reply:
Addressed